### PR TITLE
Added LibsDisguises support

### PR DESCRIPTION
**Pull request by Mwthorn**
#157

### CODE REVIEW

File: bukkit/src/main/java/com/denizenscript/depenizen/bukkit/commands/DisguiseCommands.java [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+import net.aufdemrand.denizencore.objects.Element;
+import net.aufdemrand.denizencore.objects.aH;
+import net.aufdemrand.denizencore.scripts.ScriptEntry;
+import net.aufdemrand.denizencore.scripts.commands.AbstractCommand;
+import net.aufdemrand.denizencore.utilities.debugging.dB;
+
+public class DisguiseCommands extends AbstractCommand {
+    // <--[command]
+    // @Name disguise
+    // @Syntax disguise [REMOVE/PLAYER/MOB/MISC] [target:<entity>] [type:<entity type>] [baby:true/false] [id:<number>] [data:<number>]
```
Comment (mcmonkey4eva) [inline bukkit/src/main/java/com/denizenscript/depenizen/bukkit/commands/DisguiseCommands.java]: I TYPE IN ALL CAPS BECAUSE I WANT TO FEEL LOUD

**Comment by Mwthorn** [comment]
Fixed up errors based on reviews

File: bukkit/src/main/java/com/denizenscript/depenizen/bukkit/commands/DisguiseCommands.java [hunk 2]
```diff
@@ -0,0 +1,230 @@
+package com.denizenscript.depenizen.bukkit.commands;
```
Comment (Morphan1) [inline bukkit/src/main/java/com/denizenscript/depenizen/bukkit/commands/DisguiseCommands.java]: This file should probably be more like `...bukkit.commands.libsdisguises.LibsDisguisesCommand`

File: bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/dLibsDisguise.java [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    @Fetchable("libsdisguise")
+    public static dLibsDisguise valueOf(String string, TagContext context) {
+        if (string == null) {
+            return null;
+        }
+
+        ////////
+        // Match disguise name
+
+        string = string.replace("dlibsdisguise@", "");
```
Comment (Morphan1) [inline bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/dLibsDisguise.java]: Why is this different from the `@Fetchable`? Drop the `d`

File: bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/dLibsDisguise.java [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    @Override
+    public String identifySimple() {
+        return identify();
+    }
+
+    @Override
+    public String getAttribute(Attribute attribute) {
+
+        // <--[tag]
+        // @attribute <dlibsdisguise@dlibsdisguise.type>
```
Comment (Morphan1) [inline bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/dLibsDisguise.java]: drop the starting `d` in all these too (and anywhere else it's mentioned)

File: bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/dLibsDisguise.java [hunk 5]
```diff
@@ -0,0 +1,152 @@
+package com.denizenscript.depenizen.bukkit.objects;
```
Comment (Morphan1) [inline bukkit/src/main/java/com/denizenscript/depenizen/bukkit/objects/dLibsDisguise.java]: `...bukkit.objects.libsdisguises.LibsDisguise`?

File: bukkit/src/main/java/com/denizenscript/depenizen/bukkit/commands/DisguiseCommands.java (hunk 2 above)
Comment (Mwthorn) [reply bukkit/src/main/java/com/denizenscript/depenizen/bukkit/commands/DisguiseCommands.java]: Is
com.denizenscript.depenizen.bukkit.commands.libsdisguises
fine?

**Pull request merged**